[32minfo[39m: [TestUtil]: 

*******************************************************************************
*******************************************************************************
*                                          
* Using temp dir: /tmp/hfc
*                                          
*******************************************************************************
*******************************************************************************

TAP version 13
# 

***** End-to-end flow: query chaincode *****


not ok 1 Error: Calling enrollment endpoint failed with error [Error: connect ECONNREFUSED 127.0.0.1:8054] at ClientRequest.<anonymous> (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/fabric-ca-client/lib/FabricCAClientImpl.js:883:12) at emitOne (events.js:116:13) at ClientRequest.emit (events.js:211:7) at TLSSocket.socketErrorListener (_http_client.js:387:9) at emitOne (events.js:116:13) at TLSSocket.emit (events.js:211:7) at emitErrorNT (internal/streams/destroy.js:64:8) at _combinedTickCallback (internal/process/next_tick.js:138:11) at process._tickCallback (internal/process/next_tick.js:180:9)
  ---
    operator: fail
    at: e2eUtils.tlsEnroll.then.then.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/e2eUtils.js:873:5)
    stack: |-
      Error: Error: Calling enrollment endpoint failed with error [Error: connect ECONNREFUSED 127.0.0.1:8054]
          at ClientRequest.<anonymous> (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/fabric-ca-client/lib/FabricCAClientImpl.js:883:12)
          at emitOne (events.js:116:13)
          at ClientRequest.emit (events.js:211:7)
          at TLSSocket.socketErrorListener (_http_client.js:387:9)
          at emitOne (events.js:116:13)
          at TLSSocket.emit (events.js:211:7)
          at emitErrorNT (internal/streams/destroy.js:64:8)
          at _combinedTickCallback (internal/process/next_tick.js:138:11)
          at process._tickCallback (internal/process/next_tick.js:180:9)
          at Test.assert [as _assert] (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:76:32)
          at Test.fail (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:317:10)
          at Test.bound [as fail] (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:76:32)
          at e2eUtils.tlsEnroll.then.then.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/e2eUtils.js:873:5)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
  ...
not ok 2 Error: Failed to get submitter at e2eUtils.tlsEnroll.then.then.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/e2eUtils.js:874:9) at <anonymous> at process._tickCallback (internal/process/next_tick.js:188:7)
  ---
    operator: fail
    at: e2eUtils.tlsEnroll.then.then.then.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/e2eUtils.js:897:5)
    stack: |-
      Error: Error: Failed to get submitter
          at e2eUtils.tlsEnroll.then.then.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/e2eUtils.js:874:9)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
          at Test.assert [as _assert] (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:76:32)
          at Test.fail (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:317:10)
          at Test.bound [as fail] (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:76:32)
          at e2eUtils.tlsEnroll.then.then.then.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/e2eUtils.js:897:5)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
  ...
not ok 3 Error: Failed, got error on query at e2eUtils.tlsEnroll.then.then.then.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/e2eUtils.js:898:9) at <anonymous> at process._tickCallback (internal/process/next_tick.js:188:7)
  ---
    operator: fail
    at: e2eUtils.queryChaincode.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/query.js:40:6)
    stack: |-
      Error: Error: Failed, got error on query
          at e2eUtils.tlsEnroll.then.then.then.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/e2eUtils.js:898:9)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
          at Test.assert [as _assert] (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:76:32)
          at Test.fail (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:317:10)
          at Test.bound [as fail] (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/node_modules/tape/lib/test.js:76:32)
          at e2eUtils.queryChaincode.then (/home/varun/Desktop/electron-quick-start/fabric-sdk-node/test/integration/e2e/query.js:40:6)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
  ...

1..3
# tests 3
# pass  0
# fail  3

